ci-operator/step-registry/hypershift/kubevirt/create/hypershift-kubevirt-create-chain.yaml

```diff
@@ -31,6 +31,10 @@ chain:
       oc wait --timeout=30m --for=condition=Available --namespace=clusters hostedcluster/${CLUSTER_NAME}
       echo "Cluster became available, creating kubeconfig"
       bin/hypershift create kubeconfig --name=${CLUSTER_NAME} >${SHARED_DIR}/nested_kubeconfig
+      
+      if [ "$(oc get infrastructure cluster -o=jsonpath='{.status.platformStatus.type}')" == "BareMetal" ]; then
+        exit 0
+      fi
```

Comment:
why not add the proxy to this script within this if/def?

I know it's more code, but in this case i think making this one script more complicated is actually simpler then the indirection caused by breaking out of this script earlier and executing multiple other steps for the proxy and health check.

One improvement now that this script is so large would be to break this script into a `.sh` file that's referenced by the yaml file.

Reply:
Yes, I would love to too, but AFAIK there is no ssh in this image.

Reply:
ah, ok that makes sense. i understand